src/rp2/plugin/accounting_method/hifo.py

```diff
@@ -51,14 +52,20 @@ def seek_non_exhausted_acquired_lot(
                 # The acquired lot has zero partial amount
                 continue
 
-            if selected_acquired_lot is None or selected_acquired_lot.spot_price < acquired_lot.spot_price:
+            if acquired_lot_amount > ZERO:
```

Comment:
Could you clarify this change?

Reply:
Mostly relevant for method changes across years - we want to skip already exhausted lots.

Reply:
I see, and you don't need the `selected_acquired_lot.spot_price < acquired_lot.spot_price` logic because it is implicit via heap operatons. Don't you need a similar if with the break in the LIFO case?

Reply:
This seems to be unnecessary as an equivalent check is made a few lines up. I've removed in the latest commit